cmake/modules/FindGLM.cmake

```diff
@@ -56,10 +56,6 @@ INCLUDE(FindPackageHandleStandardArgs)
 FIND_PACKAGE_HANDLE_STANDARD_ARGS(GLM DEFAULT_MSG
     GLM_INCLUDE_DIR)
 
-IF(GLM_FOUND)
-    SET(GLM_INCLUDE_DIRS "${GLM_INCLUDE_DIR}")
+SET(GLM_INCLUDE_DIRS "${GLM_INCLUDE_DIR}")
 
-    IF(NOT GLM_FIND_QUIETLY)
-        MESSAGE(STATUS "GLM_INCLUDE_DIR = ${GLM_INCLUDE_DIR}")
-    ENDIF(NOT GLM_FIND_QUIETLY)
-ENDIF(GLM_FOUND)
+mark_as_advanced(GLM_INCLUDE_DIR)
```

Comment:
Retested and the patches to findLibSndFile and findGLM are not needed.
So left them out.